plaid/requester.py

```diff
@@ -30,7 +30,15 @@ def _requests_http_request(url, method, data, timeout=DEFAULT_TIMEOUT):
 
 def http_request(url, method=None, data=None, timeout=DEFAULT_TIMEOUT):
     response = _requests_http_request(url, method, data or {}, timeout)
-    response_body = json.loads(response.text)
+    try:
+        response_body = json.loads(response.text)
+    except json.JSONDecodeError:
+        raise PlaidError.from_response({
+            'error_message': '',
```

Comment:
What are your thoughts on including either `json.JSONDecodeError` or `response.text` in `error_message`? Otherwise it's going to be tough to debug any reports of these errors, since there's very little exposed to go by.

Reply:
👍 Agreed